stable/aws-pod-identity-webhook/templates/service.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "aws-pod-identity-webhook.fullname" . }}
+  labels:
+{{ include "aws-pod-identity-webhook.labels" . | indent 4 }}
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    prometheus.io/port: "443"
+    prometheus.io/scheme: "https"
```

These annotations are incorrect.
The metrics endpoint for these pods is using `http` on port `9999`
```suggestion
    prometheus.io/port: "9999"
    prometheus.io/scheme: "http"
```